### ISSUE TITLE
Question/issue about REGEXP property constraint?

### ISSUE THREAD

**Issue by hrstoyanov** [orientechnologies/orientdb#8363]
### Java Version: 1.8

I have defined the following property:
```
--[C]reate, [U]pdate, [D]elete, [V]iew  type of action.
CREATE PROPERTY Action.type STRING  (MANDATORY TRUE, NOTNULL, REGEXP "[C|U|D|V]"); 
```
When I try to insert value C:
```
OElement action = session.newElement("Action");
action.setProperty("type","C");
```

I get this exception:
```
Caused by: com.orientechnologies.orient.core.exception.OValidationException: The field 'Action.type' does not match the regular expression 'REGEXP'. Field value is: C, record: Action{...,type:C}
```

 Why is the validator saying that my regular expression is 'REGEX' when it is clearly not  (it is [C|U|D|V])?

**Comment by hrstoyanov** [comment]
@luigidellaquila have you had a chance to look at that? I checked the demo-db schema and you guys never used REGEX constraints there?

**Comment by luigidellaquila** [comment]
Hi @hrstoyanov 

Sorry, not yet, I'll try to do it asap

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @hrstoyanov 

It was a bug, I just pushed a fix to develop branch, it will be released with v 3.0.4

Thanks

Luigi
